lib/nodes.go

```diff
@@ -85,4 +85,34 @@ var NODES = map[uint64]DeSoNode{
 		URL:   "https://nftz.zone",
 		Owner: "mvanhalen",
 	},
+	13: {
+		Name:  "Cloutible",
+		URL:   "https://cloutible.club",
+		Owner: "DawaynePerza",
+	},
+	14: {
+		Name:  "Agbegbe",
+		URL:   "https://agbegbe.org",
+		Owner: "TheParkerazzi",
+	},
+	15: {
+		Name:  "CloutingAround",
+		URL:   "https://cloutingaround.dev",
+		Owner: "TheParkerazzi",
+	},
+	16: {
+		Name:  "MediaTech",
+		URL:   "https://deso.mediatech.ventures",
+		Owner: "paulobrien",
+	},
+	17: {
+		Name:  "Mousai",
+		URL:   "https://deso.mousai.stream",
+		Owner: "marlonjm2k",
+	},
+	18: {
+		Name:  "KoalaTBooks",
+		URL:   "https://koalatbooks.com",
```

Comment:
This domain was created two weeks ago, please resubmit it later.

Reply:
@maebeam The node itself has been online for more than 30d, as per the requirement. The domain is newer, but the node was still live and processing tx's for weeks prior to the domain registration (It was used for API calls and other interactions using IP)

Not quite sure why domain registry would be used to calculate node uptime, as the metrics are entirely unrelated.

Reply:
Thanks for the clarification on uptime

Reply:
@maebeam No problem -- thanks for the merge. As per the ineligible node, I'll go head and submit another PR to remove it.